### PR TITLE
Date in system default language

### PR DESCRIPTION
**Pull request by antrrax**
Suggestion:
DatePickerDialog: Changes the property of the class to define the names of the months of the year according to the system's default language, and with the name of the month capitalized.

DateEntry: Change **dateformat** to use system default date format

### CODE REVIEW

File: src/ttkbootstrap/widgets.py [hunk 1]
```diff
@@ -95,7 +96,7 @@ class DateEntry(ttk.Frame):
     def __init__(
         self,
         master=None,
-        dateformat=r"%Y-%m-%d",
+        dateformat=locale.nl_langinfo(locale.D_FMT),
```
Comment (israel-dryer) [inline src/ttkbootstrap/widgets.py:99]: I like this idea, but unfortunately, this method is [not available on Windows](https://www.w3schools.com/pHP//func_string_nl_langinfo.asp#:~:text=The%20nl_langinfo%20%28%29%20function%20returns%20specific%20local%20information.,information%2C%20the%20nl_langinfo%20%28%29%20function%20returns%20specific%20information.). We need something that is cross-platform compatible.

**Review by israel-dryer — CHANGES_REQUESTED** [review]
Need a cross-platform solution.

**Comment by antrrax** [comment]
I returned the 'widgets.py' file to the default code.  And I kept the changes from 'dialogs.py', so the name of the month is in the default language of the system.
And the date format will be manually set in the DateEntry as it was before.

File: src/ttkbootstrap/widgets.py [hunk 2]
```diff
@@ -96,7 +95,7 @@ class DateEntry(ttk.Frame):
     def __init__(
         self,
         master=None,
-        dateformat=locale.nl_langinfo(locale.D_FMT),
+        dateformat=r"%Y-%m-%d",
```
Comment (israel-dryer) [inline src/ttkbootstrap/widgets.py:98]: What happens if you use `%x`? Does it show the correct date format for your locality?
https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-format-codes

**Review by israel-dryer — CHANGES_REQUESTED** [review]
Try the string formatting recommendation and let me know. I think this may work.

File: src/ttkbootstrap/widgets.py (hunk 2 above)
Comment (antrrax) [reply src/ttkbootstrap/widgets.py:98]: `dateformat=r"%x"`

Yes indeed, it worked. The default pt_BR format:
03/02/2022 (day/month/year)

I think you better make this change, then also modify the documentation with the necessary explanations.

**Pull request merged**